packages/fast-jss-manager-react/src/manage-jss.spec.tsx

```diff
@@ -33,6 +33,17 @@ const dynamicStyles: ComponentStyles<any, any> = {
     }
 };
 
+const stylesheetResolver: ComponentStyles<any, any> = (config: any): any => {
```

Comment:
Seems like this might be a good candidate for a function comments

Reply:
sure.